apple/Handlers/RNPanHandler.m

```diff
@@ -78,75 +87,142 @@ - (void)activateAfterLongPress
   [_gestureHandler handleGesture:self inState:RNGestureHandlerStateActive];
 }
 
-- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event
+- (void)interactionsBegan:(NSSet *)touches withEvent:(UIEvent*)event
 {
-  if ([self numberOfTouches] == 0) {
-    [_gestureHandler reset];
-  }
-#if !TARGET_OS_TV
-  if (_hasCustomActivationCriteria) {
-    // We use "minimumNumberOfTouches" property to prevent pan handler from recognizing
-    // the gesture too early before we are sure that all criteria (e.g. minimum distance
-    // etc. are met)
-    super.minimumNumberOfTouches = 20;
-  } else {
-    super.minimumNumberOfTouches = _realMinimumNumberOfTouches;
-  }
-#endif
-  [super touchesBegan:touches withEvent:event];
-  [_gestureHandler.pointerTracker touchesBegan:touches withEvent:event];
+    [_gestureHandler.pointerTracker touchesBegan:touches withEvent:event];
```

Comment:
I'm not sure why this code was at the bottom, but are you sure that order doesn't matter here? Also I can't see `[super touchesBegan:touches withEvent:event];` was it unnecessary?

Reply:
Super call was moved to another function, however it changes the order. When I tested iOS examples I didn't see any problems, but I'm still not sure if it's correct approach. Imo we should stick to the original order anyway.